src/wp-includes/block-supports/dimensions.php

```diff
@@ -29,8 +29,7 @@ function wp_register_dimensions_support( $block_type ) {
 		return;
 	}
 
-	$has_dimensions_support = block_has_support( $block_type, array( '__experimentalDimensions' ), false );
-	// Future block supports such as height & width will be added here.
+	$has_dimensions_support = block_has_support( $block_type, array( 'dimensions' ), false );
```

Comment:
Results from a wp.org search:
- 0 plugins found using `__experimentalDimensions` https://wpdirectory.net/search/01GR1JTT957N4BM1FAZAHPCQ1C
- 0 themes found using  `__experimentalDimensions` https://wpdirectory.net/search/01GR1JX1E9HM3BQQQ0GJ457B68

So I'm wondering: 

- What are the backwards-compatibility risks for users who may have a custom plugin or theme using it?
- How was `'__experimentalDimensions'` previously exposed and used?
- What is the mitigation plan for `'__experimentalDimensions'` to upgrade blocks using this setting?

Reply:
One way to mitigate:
- Check it here with an `||`.
- Throw a deprecation notice to alert to use the new `'dimensions'` instead.

Reply:
Thanks for raising this one @hellofromtonya — this is a pretty interesting case for stabilisation in that the support technically existed prior to this PR (with the name `__experimentalDimensions`) however it was a no-op, in that it didn't actually expose any controls or styles. So even if a plugin had opted in to using it, there would have been no behaviour from it. Because `min-height` is a new feature, and the dimensions controls were never exposed or promoted anywhere, I don't think we need to add any handling to deal with the former `__experimentalDimensions` name — it appears it was mostly there as a placeholder.

What do you think?

Reply:
If it was a no-op, I think we're good to leave it as is.

Reply:
Thank you @andrewserong and @ntsekouras for the explanation. 

tl;dr
No BC concern.

Let me summarize the reasoning:

* Yes, this `__experimentalDimensions` setting existed before 6.2.
* It did nothing. No functionality was attached to it. (think of it as placeholder for the future, ie today)
* If a developer is using it, there is no effect to them for removing it.

I agree then. It's not a BC concern to remove it. Thank you!